csm/core/services/audit_log.py

```diff
@@ -158,8 +165,18 @@ async def get_by_range(
         effective_limit = min(limit, max_result_window) if limit is not None else max_result_window
         effective_marker = offset if offset is not None else 0
         query_limit = QueryLimits(effective_limit, effective_marker)
+        if sort_by:
+            if component == "csm" and sort_by not in const.CSM_AUDIT_LOG_SORTABLE_FIELDS:
```

Comment:
Add constants to const file. Please verify whether the entry is already present or not.

Reply:
done @pawankumarsrivastava